src/applications/new-28-1900/tests/fixtures/mocks/user.json

```diff
@@ -24,27 +24,7 @@
         "is_veteran": true,
         "served_in_military": true
       },
-      "in_progress_forms": [
-        {
-          "form": "28-1900",
-          "metadata": {
-            "version": 0,
-            "returnUrl": "/reason-for-visit",
-            "savedAt": 1608147143961,
-            "submission": {
-              "status": false,
-              "errorMessage": false,
-              "id": false,
-              "timestamp": false,
-              "hasAttemptedSubmit": false
-            },
-            "expiresAt": 9999999999,
-            "lastUpdated": 1608147144,
-            "inProgressFormId": 1234
-          },
-          "lastUpdated": 1608147144
-        }
-      ],
+      "in_progress_forms": [],
```

Comment:
The regular mock user, used for the other Cypress tests, needs to not have an in-progress form.